core/templates/pages/teach-page/teach-page.component.html

```diff
@@ -1,137 +1,652 @@
-              [innerHTML]="'I18N_ACTION_APPLY_TO_TEACH_WITH_OPPIA' | translate" target="_blank" rel="noopener" tabindex="0">
-            </a>
-          </div>
-
-          <div class="participation oppia-about-tab-content" [ngClass]="{'oppia-about-visible-content': activeTabName === TAB_ID_PARTICIPATION}">
-            <div class="oppia-static-card-content oppia-static-card-content-narrow">
-              <h2 [innerHTML]="'I18N_TEACH_PAGE_COMMUNITY_GUIDELINES_HEADING' | translate"></h2>
-              <ol>
-                <li>
-                  <p [innerHTML]="'I18N_TEACH_PAGE_COMMUNITY_GUIDELINES_GUIDELINE_1' | translate">
-                  </p>
-                </li>
-                <li>
-                  <p [innerHTML]="'I18N_TEACH_PAGE_COMMUNITY_GUIDELINES_GUIDELINE_2' | translate">
-                  </p>
-                </li>
-                <li>
-                  <p [innerHTML]="'I18N_TEACH_PAGE_COMMUNITY_GUIDELINES_GUIDELINE_3' | translate">
-                  </p>
-                </li>
-              </ol>
-              <p [innerHTML]="'I18N_TEACH_PAGE_COMMUNITY_GUIDELINES_TEXT' | translate">
-              </p>
-
-              <h2 [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_HEADING' | translate"></h2>
-
-              <p [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_PARAGRAPH_1' | translate">
-              </p>
-
-              <ul class="oppia-publication-instructions-container">
-                <li [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_INSTRUCTION_1' | translate">
-                </li>
-                <li [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_INSTRUCTION_2' | translate ">
-                </li>
-                <li [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_INSTRUCTION_3' | translate">
-                </li>
-                <li [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_INSTRUCTION_4' | translate">
-                </li>
-                <li [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_INSTRUCTION_5' | translate">
-                </li>
-                <li [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_INSTRUCTION_6' | translate">
-                </li>
-              </ul>
-
-              <p [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_INSTRUCTIONS_PARAGRAPH_2' | translate">
-              </p>
-
-              <h2 [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_POLICY_HEADING' | translate"></h2>
-
-              <p [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_POLICY_PARAGRAPH_1' | translate">
-              </p>
-              <p [innerHTML]="'I18N_TEACH_PAGE_PUBLICATION_POLICY_PARAGRAPH_2' | translate">
-              </p>
-            </div>
-          </div>
-        </div>
+
+  <div class="oppia-teach-section-two">
+    <div class="oppia-teach-section-two-left">
+      <h3 ng-if="$ctrl.isWindowNarrow()" translate="I18N_SPLASH_BENEFITS_TITLE"></h3>
```

Comment:
Rename translate label (you can create a new one).

Reply:
yes sure ,done